.circleci/run-stackrox-ci/10-run-stackrox-ci.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+BRANCH=$1
+BUILD_NUM=$2
+
+comment_on_pr_with_ci_link() {
+    local ci_url=$1
+
+    wget --quiet https://github.com/joshdk/hub-comment/releases/download/0.1.0-rc6/hub-comment_linux_amd64
+    sudo install hub-comment_linux_amd64 /usr/bin/hub-comment
+
+    export CI_URL=$ci_url
+
+    hub-comment -template-file "${CI_ROOT}/run-stackrox-ci/ci-url-comment-template.tpl"
+}
+
+git clone git@github.com:stackrox/stackrox.git
+cd stackrox
+
+git config user.email "$GITHUB_USER_NAME"
+git config user.name "$GITHUB_USER_EMAIL"
+
+stackrox_branch="${BRANCH}_ROBOT_COLLECTOR_CI"
+if git branch | grep stackrox_branch; then
```

Comment:
Doesn't `stackrox_branch` need to be quoted and expanded here? Also, maybe we can use the quiet option for grep?
```suggestion
if git branch | grep -q "$stackrox_branch"; then
```

Reply:
Oops. Done.